proof.go

```diff
@@ -80,7 +93,8 @@ func NewEmptyRangeProof(ignoreMaxNamespace bool) Proof {
 
 // NewInclusionProof constructs a proof that proves that a namespace.ID
 // is included in an NMT.
-func NewInclusionProof(proofStart, proofEnd int, proofNodes [][]byte, ignoreMaxNamespace bool) Proof {
+func NewInclusionProof(
+	proofStart, proofEnd int, proofNodes [][]byte, ignoreMaxNamespace bool) Proof {
```

Comment:
out of curiosity, why split this across two lines?

Reply:
Wasn't meant to be broken down, fixed it now https://github.com/celestiaorg/nmt/commit/26b1ea676ac264bb2c6894efcd7a4f4aaf3924b3